apiserver/facades/client/applicationoffers/base.go

```diff
@@ -34,14 +35,15 @@ type BaseAPI struct {
 	getControllerInfo         func(context.Context) (apiAddrs []string, caCert string, _ error)
 	logger                    corelogger.Logger
 	controllerUUID            string
-	modelUUID                 model.UUID
+	modelService              ModelService
 }
 
 // checkAdmin ensures that the specified in user is a model or controller admin.
 func (api *BaseAPI) checkAdmin(
-	ctx context.Context, user names.UserTag, modelID model.UUID, backend Backend,
+	ctx context.Context, user names.UserTag, modelID model.UUID,
```

Comment:
Would you mind just renaming `modelID` to `modelUUID`?

I am not sure why we don't take the tag type here considering all callers need to decompose from a tag and then this func recomposes to a tag. This PR doesn't need to solve this problem just strange. I am probably on the git blame for this one.

Reply:
Done :)